allennlp/commands/elmo.py

```diff
@@ -0,0 +1,203 @@
+"""
+The ``elmo`` subcommand allows you to make bulk ELMo predictions.
+
+Given a pre-processed input text file, this command outputs the internal
+layers used to compute ELMo representations to a single (potentially large) file.
+
+The input file is previously tokenized, whitespace separated text, one sentence per line.
+The output is a hdf5 file (http://docs.h5py.org/en/latest/) where each
+sentence is a size (3, num_tokens, 1024) array with the biLM representations.
+
+In the default setting, each sentence is keyed in the output file by the line number
+in the original text file.  Optionally, by specifying --use_sentence_key
+the first token in each sentence is assumed to be a unique sentence key
+used in the output file.
+"""
+
+import logging
+from typing import IO, List, Iterable, Tuple
+
+import argparse
+import h5py
+import torch
+
+from allennlp.common.tqdm import Tqdm
+from allennlp.common.util import lazy_groups_of
+from allennlp.data.dataset import Batch
+from allennlp.data import Token, Vocabulary, Instance
+from allennlp.data.fields import TextField
+from allennlp.data.token_indexers.elmo_indexer import ELMoTokenCharactersIndexer
+from allennlp.nn.util import remove_sentence_boundaries
+from allennlp.modules.elmo import _ElmoBiLm
+from allennlp.commands.subcommand import Subcommand
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+DEFAULT_OPTIONS_FILE = "https://s3-us-west-2.amazonaws.com/allennlp/models/elmo/2x4096_512_2048cnn_2xhighway/elmo_2x4096_512_2048cnn_2xhighway_options.json" # pylint: disable=line-too-long
+DEFAULT_WEIGHT_FILE = "https://s3-us-west-2.amazonaws.com/allennlp/models/elmo/2x4096_512_2048cnn_2xhighway/elmo_2x4096_512_2048cnn_2xhighway_weights.hdf5" # pylint: disable=line-too-long
+
+
+class Elmo(Subcommand):
```

Comment:
We already have an `elmo.py` and `Elmo` class in `allennlp/modules/elmo.py`.  Can you rename this class to something else to avoid confusion?